### DISCUSSION TITLE
How to resolve warning in when a YAML key is faulty?

### DISCUSSION THREAD

**Discussion by adrianrudnik** [WeblateOrg/weblate#5482] [Q&A] [answered]
Just tried to transport fixed strings into a YAML based component and made the mistake to commit something like

```yaml
{
    "error-page": {
        "errors": {
            404: "Not found",
            "server": "Server error"
        }
    }
}
```

Weblate received the commit and forced a `Key not string: error-page->errors/404 (<class 'int'>)` warning that I could not get rid of.

What is the correct flow to handle those faulty commits? The only solution was to force push the fixed version to master/develop, overwriting the history. Is there any other way to solve this gracefully, like ignoring a specific commit?

**Comment by nijel** [comment, marked as answer]
First - there is no reason why numeric keys should not work now, I'm changing this in the translate-toolkit, see https://github.com/translate/translate/pull/4279

Besides that, you can always change the file in the repository...

**Reply by adrianrudnik** [reply]
Cool with the numeric keys. By "in the repository" you mean the one Weblate monitors right? I configured Weblate to pull and push changes to a code repository and no commit fixed the warning, it stayed, even after fixing the YAML file, that's why I was confused on how to resolve it. In the end only a force push the code repo and a "Maintenance" > "Reset" worked.
